ansible/deploy.yml

```diff
@@ -224,7 +224,7 @@
 
       - name: Send Datadog event
         datadog_event:
-          host: "{{ inventory_hostname }}"
+          host: "{{ public_ipv4 }}"
```

Comment:
Not sure where this variable comes from.

Reply:
I see this defined in following commit https://github.com/aeternity/infrastructure/pull/94/commits/74224e9246d3eb899b82caf3c141557e513015d1#diff-e6a0bc0f3918ce5770cb77aff9228af9R2